### PR TITLE
Update: Make the visualization toggle a full component

### PR DESCRIPTION
**Pull request by mattgraham1995**
Partials aren't recommended for use anymore so we should make this a component instead. It looks and behaves the same.

### CODE REVIEW

File: packages/reports/addon/components/visualization-toggle.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+ *    report=report
+ *    validVisualizations=validVisualizations
+ *    onVisualizationTypeUpdate=(action 'onVisualizationTypeUpdate')
+ *  }}
+ */
+import Component from '@ember/component';
+import layout from '../templates/components/visualization-toggle';
+
+export default Component.extend({
+  layout
```
Comment (jkusa) [inline packages/reports/addon/components/visualization-toggle.js]:
```js
tagName: 'span',
classNames: ['visualization-toggle']
```

File: packages/reports/addon/templates/components/visualization-toggle.hbs [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{{!-- Copyright 2018, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
+<span class='visualization-toggle'>
```
Comment (jkusa) [inline packages/reports/addon/templates/components/visualization-toggle.hbs]: We can remove this now.

Comment (mattgraham1995) [reply packages/reports/addon/templates/components/visualization-toggle.hbs]: I remembered this as I was getting in my car to leave. whoops

**Comment by mattgraham1995** [comment]
@jkusa gave the verbal approval

**Pull request merged**